internal/controllers/clusterextension_controller.go

```diff
@@ -286,10 +286,7 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 	managedObjs, _, err := r.Applier.Apply(ctx, unpackResult.Bundle, ext, objLbls, storeLbls)
 	if err != nil {
 		setStatusProgressing(ext, wrapErrorWithResolutionInfo(resolvedBundleMetadata, err))
-		// If bundle is not already installed, set Installed status condition to False
-		if installedBundle == nil {
-			setInstalledStatusConditionFailed(ext, err.Error())
-		}
+		setInstalledStatusFromBundle(ext, installedBundle, err)
```

Comment:
It's too bad we can't make lines 293-297 reuse the `setInstalledStatusFromBundle`. What would it take to do that? Seems like we'd need to construct an `installedBundle` out of the `resolvedBundle`

Reply:
I thought about that too...

Reply:
However, the `setInstalledStatusFromBundle()` function will try to avoid changing the timestamp of the condition if it's already true; so it's not constantly overwriting the timestamp (or the message) when the prior bundle was (detected as) installed.
Lines 293-297 force an update of the condition.

Reply:
No longer an issue